graphql/project_resolver.go

```diff
@@ -53,7 +53,28 @@ func (r *projectResolver) Patches(ctx context.Context, obj *restModel.APIProject
 	return &Patches{Patches: apiPatches, FilteredPatchCount: count}, nil
 }
 
+// Theme is the resolver for the theme field.
+func (r *projectBannerResolver) Theme(ctx context.Context, obj *restModel.APIProjectBanner) (BannerTheme, error) {
```

Comment:
I think you may be able to omit these resolvers by adding this model for the enum in `gqlgen.yml`

```
  BannerTheme:
    model: github.com/evergreen-ci/evergreen.BannerTheme
```

and swapping ProjectBanner and APIProjectBanner to use `evergreen.BannerTheme` for the theme type (`evergreen.BannerTheme` is defined in [`config.go`](https://github.com/evergreen-ci/evergreen/blob/main/config.go#L795-L802))

I tested it alongside your front end PR and it seemed to work, but also feel free to tell me if it sounds very wrong 🤪